src/libraries/Microsoft.PowerFx.Core/Functions/ArgPreprocessor.cs

```diff
@@ -10,9 +10,11 @@ namespace Microsoft.PowerFx.Core.Functions
     internal enum ArgPreprocessor
     {
         None = 0,
-        ReplaceWithZero = 1,
+        ReplaceBlankWithZero = 1,
         
         // This also includes ReplaceWithZero
         Truncate = 2,
+
+        ReplaceBlankWithEmptyString = 3,
```

Comment:
>ReplaceBlankWithEmptyString [](http://example.com/codeflow?start=8&length=27)

it would be good to comment why this can't just be coalesce.

Reply:
Added the comment in IRTranslator